cinder/volume/drivers/lightos.py

```diff
@@ -799,8 +799,6 @@ def delete_volume(self, volume):
             msg = ('Failed to delete LightOS volume with UUID'
                    ' %(uuid)s project %(project_name)s' % (
                        dict(uuid=lightos_uuid, project_name=project_name)))
-
-            msg = _(msg)
```

Comment:
looks fine, though would be happy to be understand/review which functions 
are invoked by operator and require the original change and which dont